samples/gazell/gzll_ack_payload_device/README.rst

```diff
@@ -0,0 +1,22 @@
+.. _gzll_ack_payload_device:
+
+Gazell ACK Payload Device
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates the basic Gazell communication and demonstrates how to send payloads and acknowledgments.
+It consists of two applications, one running on the Device and one running on the Host.
+You must follow the instructions and use of files from this page, and the :ref:`gzll_ack_payload_host` sample.
+
+Requirements
+************
+
+.. note::
+   Follow the steps from and include the :ref:`gzll_ack_payload_host` sample in addition to this sample.
```

Comment:
```suggestion
   Follow the steps from and include the :ref:`gzll_ack_payload_host` sample to this sample.
```

Reply:
Perhaps, "with this sample" would be best?

Reply:
Have applied this throughout, `with this sample.`